lib/src/invariant.dart

```diff
@@ -43,24 +45,31 @@ Position getCoord(dynamic coord) {
 /// var coords = turf.getCoords(poly);
 /// //= [[[119.32, -8.7], [119.55, -8.69], [119.51, -8.54], [119.32, -8.7]]]
```

Comment:
We should update this comment, because we don't have this namespace `turf.`, instead it's just `getCoords(...)`.
Also the result should look like this: `[[Position(119.32, -8.7), Position(...), ...]]`, instead of plain arrays